scholly/base/src/components/team/TeamPage.js

```diff
@@ -3,13 +3,24 @@
  */
 import React, { Fragment } from 'react';
 import { CardDeck, Container } from 'react-bootstrap';
-import { TEAM_MEMBERS } from '../../global/data';
+import { Helmet } from 'react-helmet';
+import { TEAM_MEMBERS, WEB_PAGES } from '../../global/data';
 import TeamMember from './TeamMember';
 
 const TeamPage = () => {
   const teamMembers = TEAM_MEMBERS;
+  const { title } = WEB_PAGES;
```

Comment:
A custom title is overkill IMO, if we know the team name, the title can just be:

`Team Page - <Team Name>`. That makes it more meaningful than a custom title I reckon.